content/docs/start/experiments/experiment-management.md

```diff
@@ -0,0 +1,139 @@
+---
+title: 'Get Started: Experiment Management'
+description:
+  'Get started with managing experiments with DVC. Manage your experiments and
+  share them with others using software engineering best practices.'
+---
+
+# Get Started: Experiment Management
+
+After having compared all the experiments, ML Practitioners still need to agree
+on which one is the best and manage the remaining candidates.
+
+<abbr>DVC Experiments</abbr> are fully compatible with Git workflows, so you can
+manage the experiments using software engineering best practices.
+
+## Sharing
+
+Unless you have enabled
+[Studio Live Experiments](/doc/studio/user-guide/projects-and-experiments/live-metrics-and-plots),
+the <abbr>DVC experiments</abbr> only exist in your repo and people can't manage
+or view them from other machines.
+
+You can share an experiment with others from your machine:
+
+<toggle>
+
+<tab title="DVC CLI">
+
+You can use `dvc exp push`, providing the names of the git remote and the
+experiment:
+
+```cli
+$ dvc exp push origin "soupy-leak"
+```
+
+People from other machines can then retrieve it using `dvc exp pull`:
+
+```cli
+$ dvc exp pull origin "soupy-leak"
+```
+
+</tab>
+
+<tab title="VSCode Extension">
+
+You can right-click on the experiment row and select the `Share to Studio`
+action:
+
+![VSCode Share to Studio](/img/vscode-sharing.gif)
+
+</tab>
+
+</toggle>
+
+<admon type="info">
+
+Learn more about
+[Sharing Experiments](/doc/user-guide/experiment-management/sharing-experiments)
+
+</admon>
+
+## Persisting
```

Comment:
Can we go with `apply` here? We have tried to put `branch` first but `apply` continues to have higher usage. It also doesn't have the awkwardness of a new branch name or leaving a dirty git workspace to clean up, and it's easier to see that everything is in your workspace. Also, I expect `branch` will seem kind of redundant once you can branch/PR experiments from Studio.

Reply:
> Can we go with apply here? We have tried to put branch first but apply continues to have higher usage.

I feel like we have gone in circles with this discussion 😅 .

My reasoning is that `exp branch` is easier to explain/understand (we are in the get started) and the less likely to fail or cause unexpected results.

> It also doesn't have the awkwardness of a new branch name 

https://github.com/iterative/dvc/pull/9211

> or leaving a dirty git workspace to clean up, and it's easier to see that everything is in your workspace.

I don't understand how that is not also true for `exp apply`.

> Also, I expect branch will seem kind of redundant once you can branch/PR experiments from Studio.

I don't understand the redundancy. 
It's like the tabs we already have where you do the same in different contexts.
When it's implemented in Studio, we can just show it in a new tab.